### DISCUSSION TITLE
希望有单条笔记的连接方便分享给别人

### DISCUSSION THREAD

**Discussion by Kaiyuan** [usememos/memos#220] [Ideas]
想分享某一条笔记，但是找不到连接。

锚点跳转的方式也可以。

**Comment by boojack** [comment]
在 `v0.4.4` 里新增里对于 Protected/Public 的链接分享按钮，会复制一个 Explore 页面的地址，类似于这样：https://demo.usememos.com/explore?memoId=1003

<img width="939" alt="image" src="https://user-images.githubusercontent.com/24653555/190834800-b41a6dd0-2fd7-4947-a711-028998e39461.png">

**Comment by flymin** [comment]
可否针对分享链接设置权限？比如主页里这条 memo 还是 private 的，但是通过分享链接可以查看。现在好像只能设置 memo 为 public 才能查看分享链接。
